doc/Configuration.md

```diff
@@ -0,0 +1,236 @@
+discussed so far apply to the webring as a whole, each member's table is unique
+to that member; the settings within do not apply to other members.
+
+Member settings:
+| Key           | Required | Type                                                        | Default |
+| ---           | ---      | ---                                                         | ---     |
+| `url`         | yes      | string (URL)                                                | none    |
+| `discord-id`  | no       | integer                                                     | none    |
+| `check-level` | no       | string (one of `none`/`off`, `online`/`up`, `links`/`full`) | `full`  |
+
+Examples (all are equivalent):
+
+- ```toml
+  [members.kian]
+  url = "https://kasad.com"
+  discord-id = 123
+
+  [members.henry]
+  url = "https://hrovnyak.gitlab.io"
+  check-level = "online"
+  ```
+
+- ```toml
+  [members]
+  kian = { url = "https://kasad.com", discord-id = 123 }
+  henry = { url = "https://hrovnyak.gitlab.io", check-level = "online" }
+  ```
+
+- ```toml
+  members.kian = { url = "https://kasad.com", discord-id = 123 }
+  members.henry = { url = "https://hrovnyak.gitlab.io", check-level = "online" }
+  ```
+
+- ```toml
+  members.kian.url = "https://kasad.com"
+  members.kian.discord-id = 123
+  members.henry.url = "https://hrovnyak.gitlab.io"
+  members.henry.check-level = "online"
+  ```
+
+### `url`
+
+The URL of the member's site.
+
+### `discord-id`
+
+The member's Discord user ID. Note that this is *not* their username, but rather
+is a number. See [this article][discord-user-id] for how to find someone's user
+ID.
+
+If unset, this member will not be notified on Discord when their site fails checks.
+
+[discord-user-id]: https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID
+
+See [the `discord` table](#discord-table) for more on the Discord integration.
+
+### `check-level`
+
+The level of check to perform on this member's site. Checks are performed on
+every site before users are sent there. E.g. if the user requests the next site
+in the ring coming from **A**, and site **B** is not online, **B** will be
+silently skipped and the user will be sent to the next passing site, e.g. **C**.
+
+The options are:
+- `off` or `none`: No checks are performed. The site is considered to always be
+  passing.
+- `up` or `online`: The site is considered to pass if it responds to HTTP
+  requests with a successful (200-299) response.
+- `full` or `links`: The site must pass the previous check, and must also
+  respond with an HTML body containing links (`<a>` elements) with the following
+  destinations as their destination (`href` attribute):
+  - `<RING_BASE_URL>`
+  - `<RING_BASE_URL>/prev` or `<RING_BASE_URL>/previous` (with optional `?host=` parameter)
+  - `<RING_BASE_URL>/next` (with optional `?host=` parameter)
+
+  If any of these links is missing, the site fails the check.
```

Comment:
The checking thing also supports having the attributes `data-phwebring="prev"|"previous"|"next"|"home"` on arbitrary elements, and those will be counted as the links.